Vagrantfile

```diff
@@ -159,6 +159,7 @@ Vagrant.configure(2) do |config|
     elk.vm.hostname = 'elk.vagrant'
     elk.vm.network "private_network", ip: "10.0.0.104"
     elk.vm.network "forwarded_port", guest: 80, host: 8084
+    elk.vm.network "forwarded_port", guest: 5601, host: 5601
```

Comment:
Okay, that sounds like a great change to add. Since this change is for fixing vagrant port issues, it all looks good to me.